### ISSUE TITLE
BUG - 2 - iOS/Android - Accessibility: Scrolling on Payment History page uses incorrect pagination

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#10780]
## What happened?

On the Payment History page, when an accessibility user uses the swiping up/down gesture to navigate a page via pagination, the page reads out "page 1 of 3" and "page 3 of 3", but does not appear to have a "page 2 of 3" available.

## Specs:

- Device: Samsung S24, iPhone 15 plus
- OS: Latest
- User Account: DEMO
- Accessibility State: Screen reader ON, default settings

## Steps to Reproduce

1. Make sure the device screen reader is ON. Log into the app as the DEMO user.
2. Access the Payment History page.
3. Using the default screen reader gesture (usually a vertical swipe with 3 fingers), swipe to navigate down to the bottom of the page, then back to the top of the page.
4. Observe - The screen reader will announce "page 1 of 3" and "page 3 of 3", but there is no pagination for "page 2 of 3".

## Desired behavior

The screen reader should have announced "page 1 of 2" and "page 2 of 2" if there was no need for additional 3rd 'page' of pagination.

## Acceptance Criteria

When using a screen reader to navigate a page via pagination, the screen reader should announce the correct sequential pagination when navigate a page.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High **Low** 
- Frequency: **High** Low 

- 3 - Low
**- 2 - High**
- 1 - Critical

## Screen shot(s) and additional information

https://github.com/user-attachments/assets/3147bb03-2690-4e34-9f4c-b0df359fc3a8

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by matthew-df** [comment]
The cause of the this bug comes from VoiceOver itself. Depending on the size of the scrollable area combined with the size of the screen the page navigation gesture will 'skip over' a page but not the actual content. This is not always an issue but depends on the size of the list. This can be replicated in other apps as well that have where the scroll areas can expand and do not fetch data on scroll. Tested on the iOS Weather, Notes and Develop apps. Here is an example of Developer:

https://github.com/user-attachments/assets/bf612b11-a298-4e19-8c2c-1a40b2b901a1
